litebird_sim/observations.py

```diff
@@ -5,7 +5,13 @@
 import numpy as np
 
 from .distribute import distribute_evenly
-from .scanning import Spin2EclipticQuaternions, all_compute_pointing_and_polangle
+
+# NOTE: remove these imports after the deprecated methods have been removed
```

Comment:
Wait, why should these imports be removed? They are referring to the *functions*, not to the methods of the `Observation` class, right?

Reply:
Yes, but once the deprecated methods are removed from Observation, they are not necessary anymore in this class. Right?

Reply:
Ok, but the comment is misleading, as it was placed above the exported *functions*, not the methods.

Reply:
Ah ok, now I see your point, I'll clarify.